tests/foreman/ui/test_ansible.py

```diff
@@ -519,6 +519,43 @@ def test_positive_assign_and_remove_ansible_role_to_hostgroup(
             session.hostgroup.delete(name)
             assert not target_sat.api.HostGroup().search(query={'search': f'name={name}'})
 
+    @pytest.mark.tier3
+    def test_positive_non_admin_access(
```

Comment:
can we change this test name to something meaningful?

Reply:
The test name has been changed.